tests/unit/runtime/zero/test_offload_states.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Need unit test to verify that 
1. tensor.device == cpu after `offload_states`
2. tensor.device == accelerator after `reload_states`

Reply:
Added validation